configure/configure.sh

```diff
@@ -10,29 +10,29 @@
 #
 
 #for now, gotta be macOS 10.12+
-OSVers=`sw_vers -productVersion`
-if [[ ${OSVers:3:2} -lt 12 ]]; then
-echo "\nERROR:" $OSVers "is currently unsupported"
-echo "LuLu requires macOS 10.12+\n"
-exit -1
+OSVers="$(sw_vers -productVersion)"
+if [ "${OSVers:3:2}" -lt 12 ]; then
+    echo "\nERROR: ${OSVers} is currently unsupported"
+    echo "LuLu requires macOS 10.12+\n"
+    exit -1
 fi
 
 #gotta be root
-if [ "$EUID" -ne 0 ]; then
-echo "\nERROR: must be run as root\n"
-exit -1
+if [ "${EUID}" -ne 0 ]; then
+    echo "\nERROR: must be run as root\n"
+    exit -1
 fi
 
 
-if [ "$1" == "-install" ]
+if [ "${1}" == "-install" ]
 then
     echo "installing"
 
     #move to unzipped directory
-    cd "`dirname "$0"`"
+    cd "$(dirname "${0}")" || exit -2
```

Comment:
The `||` is a [`shellcheck`](https://github.com/koalaman/shellcheck) recommendation. It prevents problems if the `cd` can’t be done (extremely unlikely, but no reason to not do it).